modules/cnv-importing-vmware-vm.adoc

```diff
@@ -1,67 +1,59 @@
 // Module included in the following assemblies:
 //
-// * cnv/cnv_users_guide/cnv-create-vms.adoc
+// * cnv/cnv_users_guide/cnv-importing-vmware-vm.adoc
+[id='Importing_vmware_vm_or_template_{context}']
+= Importing a VMware virtual machine or template
 
-= Importing the VMware virtual machine or template
+You can import a VMware virtual machine or template using the virtual machine wizard.
 
 .Prerequisites
 
-* The VMware virtual machine is powered off.
+* You must create a VDDK image, push it to a private container repository, and add it to the `v2v-vmware` Configmap.
+* You must power off the VMware virtual machine before importing it.
 
 .Procedure
 
-. Click *Workloads* -> *Virtual Machines* from the side menu.
-. Click *Create Virtual Machine* and select *Create with Wizard*.
-. Perform the following procedure in the *Basic Settings* screen:
-.. Click *Provision Source* and select *Import*.
-.. Click *Provider* and select *VMware*.
-.. Click *vCenter Instance* and select *Connect to New Instance* or a saved vCenter instance.
-+
-* If you select *Connect to New Instance*, provide values in the following fields and click *Check and Save*:
+. In the {CNVProductName} web console, click *Workloads* -> *Virtual Machines*.
+. Click *Create* and select *Import with Wizard*.
+. In the *General* screen, perform the following steps:
+.. Select *VMware* from the *Provider* list.
+.. Select *Connect to New Instance* or a saved vCenter instance from the *vCenter instance* list.
 
-** *vCenter Hostname*
-** *vCenter User Name*
-** *vCenter Password*
-+
-The wizard connects to the vCenter instance and saves the credentials.
+.. Select a virtual machine or a template to import.
 +
-If you clear the *Save as New vCenter Instance* check box and click *Check*, the wizard connects to the vCenter instance without saving the credentials.
+The *Name* and *Operating System* are filled in automatically with the attributes of the selected virtual machine or template.
 
-* If you select a saved vCenter instance, the wizard connects to it by using the saved credentials.
+.. Select an existing flavor or *Custom* from the *Flavor* list.
+.. Select a *Workload Profile* and then click *Next*.
 
-.. Click *VM to import* and select a virtual machine or template.
-+
-The *Name* and *Operating System* are filled in automatically with the attributes of the selected virtual machine or template.
+. In the *Networking* screen, perform the following steps:
 
```

Comment:
In order to add a Network interface, need to click the "Add Network Interface" button, in the *Networking* screen.

Reply:
Is adding a network interface a requirement or an option?

Reply:
An option. It is not something new. We had this option on CNV-2.0 too.

Reply:
If adding an interface is not a requirement, it does not need to be documented in the procedure. We don't normally document all options. Also, the user can probably figure out what an "Add network interface" button does. :-)